frontend/src/components/search/PermitResultsTable.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Table, Row, Col, Divider } from "antd";
+import PropTypes from "prop-types";
+import Highlight from "react-highlighter";
+import { Link } from "react-router-dom";
+import * as router from "@/constants/routes";
+
+/**
+ * @class  PermitResultsTable - displays a table of mine search results
+ */
+
+const propTypes = {
+  header: PropTypes.string.isRequired,
+  highlightRegex: PropTypes.objectOf(PropTypes.regexp).isRequired,
+  searchResults: PropTypes.arrayOf(PropTypes.object).isRequired,
+};
+
+const defaultProps = {};
+
+export const PermitResultsTable = (props) => {
+  const columns = [
+    {
+      title: "Permit Guid",
+      dataIndex: "permit_guid",
+      key: "permit_guid",
+      render: (text, record) => [
+        <Row>
+          <Col span={24}>
+            <Link
+              to={router.MINE_SUMMARY.dynamicRoute(record.mine_guid, "permit")}
+              style={{ fontSize: "1.25rem" }}
```

Comment:
Are these links supposed to be larger than other links inside tables? 
Links should all be the same size, I fear we are going to run into more inconsistencies across components.

Reply:
I thought the wire-frames depicted them larger then the other text, although double checking now they don't. Comparing, I do think it's easier to scan the results with the names being larger but I'll change it for consistency.

Reply:
Changed